src/VirtualDesktop/Interop/Build22621/.interfaces/IVirtualDesktopNotification.cs

```diff
@@ -0,0 +1,93 @@
+
+        void VirtualDesktopDestroyFailed(IObjectArray p0, IVirtualDesktop pDesktopDestroyed, IVirtualDesktop pDesktopFallback);
+
+        void VirtualDesktopDestroyed(IObjectArray p0, IVirtualDesktop pDesktopDestroyed, IVirtualDesktop pDesktopFallback);
+
+        void VirtualDesktopMoved(IObjectArray p0, IVirtualDesktop pDesktop, int nIndexFrom, int nIndexTo);
+
+        void VirtualDesktopRenamed(IVirtualDesktop pDesktop, HString chName);
+
+        void ViewVirtualDesktopChanged(IApplicationView pView);
+
+        void CurrentVirtualDesktopChanged(IObjectArray p0, IVirtualDesktop pDesktopOld, IVirtualDesktop pDesktopNew);
+
+        void VirtualDesktopWallpaperChanged(IVirtualDesktop pDesktop, HString chPath);
+        
+        IVirtualDesktop VirtualDesktopSwitched();
+        
+        IVirtualDesktop RemoteVirtualDesktopConnected();
+    }
+
+    internal class VirtualDesktopNotification : VirtualDesktopNotificationService.EventListenerBase, IVirtualDesktopNotification
+    {
+        public void VirtualDesktopCreated(IObjectArray p0, IVirtualDesktop pDesktop)
+        {
+            this.CreatedCore(pDesktop);
+        }
+
+        public void VirtualDesktopDestroyBegin(IObjectArray p0, IVirtualDesktop pDesktopDestroyed, IVirtualDesktop pDesktopFallback)
+        {
+            this.DestroyBeginCore(pDesktopDestroyed, pDesktopFallback);
+        }
+
+        public void VirtualDesktopDestroyFailed(IObjectArray p0, IVirtualDesktop pDesktopDestroyed, IVirtualDesktop pDesktopFallback)
+        {
+            this.DestroyFailedCore(pDesktopDestroyed, pDesktopFallback);
+        }
+
+        public void VirtualDesktopDestroyed(IObjectArray p0, IVirtualDesktop pDesktopDestroyed, IVirtualDesktop pDesktopFallback)
+        {
+            this.DestroyedCore(pDesktopDestroyed, pDesktopFallback);
+        }
+
+        public void VirtualDesktopMoved(IObjectArray p0, IVirtualDesktop pDesktop, int nIndexFrom, int nIndexTo)
+        {
+            this.MovedCore(p0, pDesktop, nIndexFrom, nIndexTo);
+        }
+
+        public void VirtualDesktopRenamed(IVirtualDesktop pDesktop, HString chName)
+        {
+            this.RenamedCore(pDesktop, chName);
+        }
+
+        public void ViewVirtualDesktopChanged(IApplicationView pView)
+        {
+            this.ViewChangedCore(pView);
+        }
+
+        public void CurrentVirtualDesktopChanged(IObjectArray p0, IVirtualDesktop pDesktopOld, IVirtualDesktop pDesktopNew)
+        {
+            this.CurrentChangedCore(pDesktopOld, pDesktopNew);
+        }
+
+        public void VirtualDesktopWallpaperChanged(IVirtualDesktop pDesktop, HString chPath)
+        {
+            this.WallpaperChangedCore(pDesktop, chPath);
+        }
+        
+        public IVirtualDesktop VirtualDesktopSwitched()
+        {
+            this.VirtualDesktopSwitched();
+        }
+
+        public IVirtualDesktop RemoteVirtualDesktopConnected()
+        {
+            this.RemoteVirtualDesktopConnected();
```

Comment:
This fails to build because it doesn't return.
Should be return this.RemoteVirtualDesktopConnected();